src/lwc/geFormWidgetAllocation/geFormWidgetAllocation.js

```diff
@@ -153,79 +161,183 @@ export default class GeFormWidgetAllocation extends LightningElement {
         this.init();
     }
 
+    /**
+     * @description Contains the load behavior for the widget. The load behavior depends on the format of the data
```

Comment:
Also a little nitpicky :) but this comment and the other new comments/descriptions below could be formatted to be a little narrower ( I think we try to break at 120 char wide?)

Reply:
I think this is actually under 120 chars actually. this is a screenshot from Intellij and that gray line is at 120 chars in the column.
![Screen Shot 2020-03-16 at 7 04 53 PM](https://user-images.githubusercontent.com/58006245/76806923-1e1c4000-67b9-11ea-93cc-b6fa2c7b6ed7.png)